packages/router-generator/tests/generator/nested-verboseFileRoutes-true/tests.test-d.ts

```
@@ -10,9 +10,10 @@ import {
   useRouteContext,
   useSearch,
 } from '@tanstack/react-router'
+import type { FileRoutesByPath, MakeRouteMatch } from '@tanstack/react-router'
```

ESLint

[error] `@tanstack/react-router` type import should occur after import of `./routeTree.gen`

(import/order)